docs/user-guide/environment-variables.md

```diff
@@ -59,7 +59,12 @@ function:
 ```javascript
 let foo = env.get("FOO");
 ```
-
+### Template node
+Beginning with [version 3.0](https://nodered.org/blog/2022/07/14/version-3-0-released#environment-variables-in-the-template-node
+), the `template` node can access environment variables using the syntax:
```

Comment:
```suggestion
*Since [Node-RED 3.0](https://nodered.org/blog/2022/07/14/version-3-0-released#environment-variables-in-the-template-node)*

The `template` node  can access environment variables using the syntax:
```